src/generate_aian_var.ado

```diff
@@ -42,66 +40,53 @@ Website
 
 
 - - -
-
-This help file was dynamically produced by 
-[MarkDoc Literate Programming package](http://www.haghish.com/markdoc/) 
+{it:This help file was dynamically produced by {browse "http://www.haghish.com/markdoc/":MarkDoc Literate Programming package}.}
 ***/
 
 
+* capture program drop generate_race_var
+
 program generate_race_var
```

Comment:
Stata not successfully loading. First error message is `command generate_aian_var not defined by generate_aian_var.ado`. Subsequent error messages say `program generate_race_var already defined`.

Reply:
Silly me misnamed the program in the `.ado` file.